_posts/2014-06-06-paid_time_off.markdown

```diff
@@ -21,98 +19,3 @@ These endpoints handle paid time off for an employee. Currently only 'Vacation H
 | `accrual_balance`             | String            |           |    X     |    0    | how many accrual_units have been accrued
 | `maximum_accrual_balance`     | String            |           |    X     | null    | the maximum accrual allowed. A null value signifies no maximum accrual
 | `paid_at_termination`         | Boolean           |     X     |          |         | whether to pay out the accrual_balance to the employee upon their termaination
-
-## Get paid time off for an employee
-
-**HTTP Method**: `GET`
```

Comment:
In your zenpayroll PR, you only deleted the `PUT` and `POST`. If you were planning on leaving the read endpoints, you should add the relevant documentation back, and add back the navigation links. If you want to get rid of the endpoint altogether, you need to update your zenpayroll PR to delete the controller and routes.

I'm fine either way. When we looked at logentries it seems no one called the `paid_time_off` endpoint via any HTTP verb, so we might want to delete the endpoint altogether to lower our API surface area.